.env.example

```diff
@@ -14,3 +14,4 @@ KEYCLOAK_CONFIG_PATH = "keycloak-dev.json"
 AUTH_API_URL = "https://auth-api-dev.apps.silver.devops.gov.bc.ca/api/v1/"
 STATUS_API_URL = "https://status-api-dev.apps.silver.devops.gov.bc.ca/api/v1/"
 AUTH_LD_CLIENT_ID = "5db9da115f58e008123cd783"
```

Comment:
This is the LaunchDarkly client-side ID for project `sbc-auth`. You should probably configure up your own project.

Reply:
Leave this as the "dev environment" value in this file, so that locally you get the Dev flags by default.

Reply:
Change this ID to `5ddc05785933410824ecf5ab`.